option-parser.js

```diff
@@ -16,27 +17,74 @@ const defaultTargets = [
 const pkg = pkgRoot();
 let enact = pkg.meta.enact || {};
 
-function gentlyParse(file) {
+const capitalize = name => name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
Something was niggling at the back of my head when I saw this code before but I ignored it at the time.  `slice()` is not unicode aware so this could end up with some really bad interactions where the first character is uppercased and then the unicode character is sliced up and half is appended to the character.  We'll need a better capitalize function.

Reply:
name contains a file name which is mostly likely in the ASCII range, so you probably don't need to worry about the i18n part

Reply:
I guess if we're going to abandon the idea of i18n then we might as well use `name[0]` instead of `charAt(0)`.